docs/DeveloperGuide.md

```diff
@@ -374,11 +371,79 @@ otherwise)
 
 * 3a. The given index is invalid.
 
-    * 3a1. AddressBook shows an error message.
+    * 3a1. ClickConnect shows an error message.
 
       Use case resumes at step 2.
 
-*{More to be added}*
+**System**: ClickConnect
+
+**Use case**: UC02 - See usage instructions
+
+**Actor**: User
+
+**MSS**
+
+1. User opens ClickConnect application
+2. User selects option to view usage instructions from main menu
```

Comment:
Would this be better if phrased as "User requests to view usage instructions"? This way, we leave the implementation of the UI interactions to the developer.

Reply:
makes sense. have made the changes. thanks!